stdlib/public/core/SmallString.swift

```diff
@@ -108,27 +99,21 @@ extension _SmallString {
   @inlinable
   internal var isASCII: Bool {
     @inline(__always) get {
-      return discriminator.smallIsASCII
+      return _StringObject.getSmallIsASCII(fromRaw: rawDiscriminatedObject)
     }
   }
 
   // Give raw, nul-terminated code units. This is only for limited internal
   // usage: it always clears the discriminator and count (in case it's full)
-  @inlinable
+  @inlinable @inline(__always)
   internal var zeroTerminatedRawCodeUnits: RawBitPattern {
-    @inline(__always) get {
-      return (
-        self._storage.0,
-        self._storage.1 & _StringObject.Nibbles.largeAddressMask)
-    }
+    return (self._storage.0, self._storage.1 & 0x00FF_FFFF_FFFF_FFFF)
```

Comment:
Is this mask correct, or should it be `0x0FFF_` ...? Is there a reason not to use the named mask any longer here?

Reply:
Small string's contents have now diverged from the addresses. Small string uses the second-highest nibble for the count, while large strings consider that to be part of the address (or whatever future HW or OSes want).